spec/models/user_spec.rb

```diff
@@ -569,4 +569,61 @@
       end
     end
   end
+
+  describe '.condition_response' do
```

Comment:
This doesn't belong here. Doesn't MemberExpireAlert take care of this now?

Reply:
That's been removed.

Reply:
<sigh> I thought it was.  My code is clearly screwed up.  argh